src/test/java/io/neonbee/LauncherTest.java

```diff
@@ -46,26 +51,18 @@ void throwErrorIfWorkingDirValueIsEmpty() {
     }
 
     @Test
-    @DisplayName("should throw error, if instance-name is empty")
+    @DisplayName("should throw an error, if instance-name is empty")
     void throwErrorIfInstanceNameIsEmpty() {
-        args = new String[] { "-cwd", tempDirPath.toAbsolutePath().toString(), "-name", "" };
+        args = new String[] { "-cwd", workDir, "-name", "" };
         IllegalArgumentException exception =
                 assertThrows(IllegalArgumentException.class, () -> parseOptions(INTERFACE.parse(List.of(args))));
```

Comment:
nit: you could also introduce a method to do smth like this to avoid the line break since this appears 6-7 times:
```
    @Test
    @DisplayName("should throw an error, if instance-name is empty")
    void throwErrorIfInstanceNameIsEmpty() {
        args = new String[] { "-cwd", workDir, "-name", "" };
        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, this::parseOptionsArgs);
        assertThat(exception.getMessage()).isEqualTo("instanceName must not be empty");
    }

```

Reply:
I guess this won't work, as `parseOptionsArgs` takes an argument.

Reply:
it works, look where args is defined